### PR TITLE
Add invoice number view tree

### PR DESCRIPTION
**Pull request by archetipo**
This PR contains the porting form [addons-vauxoo/invoice_number_view_tree](https://github.com/Vauxoo/addons-vauxoo/tree/8.0/invoice_number_view_tree),
and it was preserved the history of commits

### CODE REVIEW

File: invoice_number_view_tree/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+{
+    "name": "Invoice number",
+    "version": "1.0",
+    "author": "Vauxoo",
+    "category": "Localization",
+    "description": """""",
```
Comment (tafaRU) [inline invoice_number_view_tree/__openerp__.py]: Please remove this key

File: invoice_number_view_tree/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        "account"
+    ],
+    "demo": [],
+    "data": [
+        "view/invoice_number.xml"
+    ],
+    "test": [],
+    "js": [],
+    "css": [],
+    "qweb": [],
```
Comment (tafaRU) [inline invoice_number_view_tree/__openerp__.py]: Please remove empty keys.

File: invoice_number_view_tree/README.rst [hunk 3]
```diff
@@ -0,0 +1,32 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Invoice number:
+===============
+
+supplier_invoice_number field added in the tree and search view by
+inheritance
+
```
Comment (tafaRU) [inline invoice_number_view_tree/README.rst:9]: Please add `Bug Tracker` section, see https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst as reference.

File: invoice_number_view_tree/__init__.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# -*- encoding: utf-8 -*-
```
Comment (tafaRU) [inline invoice_number_view_tree/__init__.py]: This file is empty so it can be deleted.

File: invoice_number_view_tree/__init__.py [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+#
+#    Module Writen to OpenERP, Open Source Management Solution
```
Comment (tafaRU) [inline invoice_number_view_tree/__init__.py]: s/writen/written

File: invoice_number_view_tree/__openerp__.py [hunk 6]
```diff
@@ -0,0 +1,43 @@
+# -*- encoding: utf-8 -*-
+#
+#    Module Writen to OpenERP, Open Source Management Solution
```
Comment (tafaRU) [inline invoice_number_view_tree/__openerp__.py]: Same as above

**Comment by pedrobaeza** [comment]
Modules should be named _account_invoice_supplier_number_info_ or similar, but it's a bit light to be a module itself. I would suggest to propose this on OCB (it's only a view change, not DB layout change), and in Odoo master to be included in next version.

:-1:

**Comment by archetipo** [comment]
thanks @pedrobaeza.
ok then I close it and proceed as you suggest .

**Comment by eLBati** [comment]
> I would suggest to propose this on OCB (it's only a view change, not DB layout change), and in Odoo master to be included in next version

Hi @pedrobaeza 
It seems `supplier_invoice_number` is not present in master: https://github.com/odoo/odoo/blob/master/addons/account/models/account_invoice.py

Anyway, as this functionality is not a bug fix, I think it is more suitable for a small module, rather than an OCB patch

**Pull request closed without merging**